### PR TITLE
Add docs for return values from checkViewport

### PR DESCRIPTION
**Pull request by ediblecode**
The existing docs only showed usage (parameters) of the visual regression
service methods, but didn't say anything about their return values. These
return values can be useful when asserting e.g.: `expect(result.isExactSameImage, errorMessage).to.be.true;` so should ideally be documented.

## Proposed changes

Documents the return values from `checkViewport`

## Types of changes

Documentations

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

n/a

### CODE REVIEW

**Comment by christian-bromann** [comment]
@ediblecode can you make sure to make a pull request for the service too?

**Comment by ediblecode** [comment]
Yes, I will do. There's no site for that right, just as part of the readme?

**Comment by christian-bromann** [comment]
> There's no site for that right, just as part of the readme?

Exactly

File: docs/guide/services/visual-regression.md [hunk 1]
```diff
@@ -144,6 +144,35 @@ available:
...
+            misMatchTolerance: 0.01
+        })
+    }
+  };
+  ```
+
+* **isSameDimensions**  `Boolean` <br>
+  True if the *taken* and *reference* images have the same dimensions; otherwise false.
+
+* **isExactSameImage**  `Boolean` <br>
```
Comment (emilyrohrbough) [inline docs/guide/services/visual-regression.md]: This is great! Our project uses these values. I would make note that `isExactSameImage` can return `true`, even if `isSameDimensions` is false. We saw this happen when a element (for example a div) grew in size, but the initial screenshot's content and the new (larger) screenshot matched 100%.

Comment (ediblecode) [reply docs/guide/services/visual-regression.md]: Thanks @emilyrohrbough I'll add this in

**Comment by christian-bromann** [comment]
@ediblecode could you please make a new PR against the v4 branch? Thanks

**Comment by ediblecode** [comment]
@christian-bromann sure, will do when I get chance - probably week after next

**Pull request closed without merging**